include/librealsense2/dds/topics/control/control.idl

```diff
@@ -0,0 +1,24 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+module librealsense
+{
+    module dds
+    {
+        module topics
+        {
+            module raw
+            {
+                module device
+                {
+                    struct control
+                    {
+                        unsigned short id; // See "librealsense::dds::topics::control::msg_type"
+                        unsigned long size;
+                        sequence<octet> raw_data;
```

Comment:
As opposed to non-raw? What makes it raw? It's just `data`, no?

Reply:
So this is unbound, variable-size? Does this place any limitations for shared-memory usage, for example?

Reply:
The `raw_data` means that this is a (unknown) sequence of bytes to be sent, this against the specified structured data.
About the data size yes, it is unbound currently (like notification topic) like discussed might need to be changed in the future

Reply:
So `data` is somehow structured? I'm saying there is no other data here -- so just call it `data`, the `raw` does not add anything. If anything it makes me look for some other data, just like `raw::device_info` makes me look for the real device_info.

Reply:
Done